### ISSUE TITLE
Not working on Next.js

### ISSUE THREAD

**Issue by pablocompagni-contractorvp** [minwork/react#9]
I'm getting the following error trying to use this on a Next.js app, I would assume because part of the code is running server side and the library is not correctly considering that possibility:

```ReferenceError: window is not defined ```

Thanks

**Comment by raskyer** [comment]
I can confirm it's working on 3.1.2-rc.1

**Comment by minwork** [comment]
Thanks @raskyer! I've published `v3.1.2` with the fix and also added test to ensure it will be working in SSR context in the future

**Comment by szamanr** [comment]
for me the app runs fine, but the event is not working. when i do a long press, i get a console error: ` Uncaught ReferenceError: global is not defined`.

my code:

```
  const toggleDevModeWhenLongPressedThreeTimes = useLongPress(toggleDevMode, {
    onCancel: switchBetweenDevModes,
  });
```

**Comment by minwork** [comment]
@szamanr Fixed in `v3.1.3`
